docs/site-example/site.conf

```diff
@@ -190,4 +190,9 @@
   --     show_altitude = false,
   --   },
   -- },
+
+  -- define if the contact field is obligatory
```

Comment:
It seems to be not consistent how to highlight optional settings, so maybe this is fine as it is.